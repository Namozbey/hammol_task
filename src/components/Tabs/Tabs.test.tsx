import React from "react";
import { render } from "@testing-library/react";
import { Tabs } from ".";

const options = [
  { label: "All", value: "all" },
  { label: "Docs", value: "doc" },
];

describe("Testing Tabs", () => {
  it("Snapshots", () => {
    const { asFragment } = render(
      <Tabs current="doc" options={options} extra={<button>Extra</button>} />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
