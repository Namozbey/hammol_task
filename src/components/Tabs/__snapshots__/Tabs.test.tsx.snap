// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Tabs Snapshots 1`] = `
<DocumentFragment>
  <div
    class="sc-dkzDqf cJLwIb"
  >
    <div
      class="sc-hKMtZM cMzcUY"
    >
      <div
        class="sc-bczRLJ joHccK"
        value="all"
      >
        <p
          class="sc-gsnTZi ZIjmp"
        >
          All
        </p>
      </div>
      <div
        class="sc-bczRLJ cPXfuW"
        value="doc"
      >
        <p
          class="sc-gsnTZi ZIjmp"
        >
          Docs
        </p>
      </div>
    </div>
    <button>
      Extra
    </button>
  </div>
  <div
    class="sc-eCYdqJ jhWBFm"
  />
</DocumentFragment>
`;
